src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -897,14 +668,11 @@ class OperatorStore extends BaseStore {
             while (iterations <= MyAccountStore.CHECK_UPDATES_MAX_ITERATIONS) {
```

Comment:
@IlyaVi , @axelrod-blox This code

```
let iterations = 0;
            while (iterations <= MyAccountStore.CHECK_UPDATES_MAX_ITERATIONS) {...

iterations += 1;
if (changed) {...

```

is basically a duplicatd 6 times in Validator and operator stores, it can be generalized (like you did in src/services/events.service.ts) and moved to a common entity. Looks like maybe we should have some base entity for operator and validators that extends BaseStore, and move such things there, something like that.